### ISSUE TITLE
Simplify Hellochain Tutorial

### ISSUE THREAD

**Issue by ghost** [cosmos/sdk-tutorials#221]
I landed on a tutorial:

https://tutorials.cosmos.network/hellochain/tutorial/00-intro.html

It starts a bitoverwhelming, like this:

```sh
mkdir -p $GOPATH/src/github.com/{ .Username }/hellochain
cd $GOPATH/src/github.com/{ .Username }/hellochain
git init
```

which is not very convenient (possibly google's go fault, but still).

## Expected

Simplify this in the way:

### Prerequisites

* Go
* git

### Setup

Within your usual project location:

```sh
mkdir hellochain
cd hellochain
git init
```

**Comment by marbar3778** [comment]
we need to rewrite the tutorial to use our scaffolding tool. I or my colleagues will spend time on this in the coming days

**Comment by lazaridiscom** [comment]
> we need to rewrite the tutorial to use our scaffolding tool. 

The Idea of "doing everything by hand" is a good one. Personally, I don't like scaffolding very much, as it avoids the need to do decent reusable design. Though "scaffolding" does not exclude "good framework design", if done right.

Note that as a newcomer, I have the benefit of seeing barriers which the existent team does not see anymore. I'll provide a little bit more feedback within the next 48h.


## Possible Spawning Issue

* Allow for easy "project" creation within any location on filesystem.

**Comment by lazaridiscom** [comment]
> we need to rewrite the tutorial to use our scaffolding tool. I or my colleagues will spend time on this in the coming days

I reached https://tutorials.cosmos.network/hellochain/tutorial/12-make.html, i am on Windows 10, and need here to abort (too lazy to work out the "missing make" barrier). Possibly I retry on an virtual machine later.

Scaffolding should do it, i guess, but a "prerequisite" section should be added, too.

As an example for "straight to the point", see this EOS starter:

https://github.com/lazaridiscom/dapp-platform/tree/master/eos

You must be clear about your target-audience (e.g.: professional devs & evaluators, casual tech-kids which want to play with crypto => all is fine, just state it. I tend to say that addressing the pro's should be prioritized)

**Comment by lazaridiscom** [comment]
@marbar3778 , re "we need to rewrite the tutorial to use our scaffolding tool", it seems not necessary, I noticed that code is in the repo, so if one wishes to do it faster => clone repo, go through tuturial and be happy that files are there.

Will most possibly try this tomorrow (on Windows WSL, see  #228), and comment.

**Comment by lazaridiscom** [comment]
Looks like it works fine on WSL.

Now, the difficult part is to evolve the tutorial and the related code, thus this becomes simple even for non-dev users.

**Comment by hschoenburg** [comment]
`mkdir -p $GOPATH/src/github.com/{ .Username }/hellochain` this command is a simplifying step. It helps the user avoid potential complictions caused by GOPATH and is a standard setup step for new go projects. `hellochain` is not intended to be an introduction to golang.

**Comment by marbar3778** [comment]
GOPATH is not a requirement of the go version we specify anymore. The hellochain tutorial should be rewritten to use scaffold and deprecate GOPATH

**Comment by lazaridiscom** [comment]
Yes, GOPATH is "old news", which made things simpler. Additionally, I managed to make it simple enough, even without scaffolding (just by adding some trivial steps).  Will post you something within 24h (should be easily adoptable for other tutorials, too).

**Comment by lazaridiscom** [comment]
See #238 for how simple this can get.

After this, it becomes more about #223 (which remains still closed, despite my complaint within #236)
